topics/statistics/tutorials/classification_regression/tutorial.md

```diff
@@ -30,13 +30,21 @@ contributors:
 # Introduction
 {:.no_toc}
 
-[Supervised learning](https://en.wikipedia.org/wiki/Supervised_learning) methods in machine learning have targets/classes/categories defined in the datasets. These targets can either be discrete values (integers) or real-values (continuous). When the targets are discrete, the learning task is known as classification. These discrete values are called classes or categories. When the targets are real-values, the task becomes regression. Classification is assigning a category to each sample in the dataset. Regression assigns a real-valued output or target to each sample. In figure [1](#figure-1), the line is a boundary which separates blue balls from violet ones. The task of a classification method is to learn this boundary which can be used to differentiate between unseen balls. The line is the decision boundary which determines the category of a new ball.
+Machine learning is a combined field of computer science, mathematics and statistics to create a predictive model by learning patterns in a dataset. The dataset may have an output field which makes the learning process supervised. The [supervised learning](https://en.wikipedia.org/wiki/Supervised_learning) methods in machine learning have outputs/targets/classes/categories defined in the datasets. These targets can either be discrete (integers) or real (continuous). When the targets are discrete, the learning task is known as classification. Thus, the classification is assigning a class/target to each sample (each row is a sample) in the dataset. The algorithms which are used for this learning task are called classifiers. When the targets are continuous, the learning task is called regression and the algorithms which are used for this task are called regressors. We will go through classification first and look at regression later in this tutorial.
```

Comment:
Machine learning is a subset of artificial intelligence (AI) that provides machines the ability to automatically learn from data without being explicitly programmed. Machine learning is a combined field of computer science, mathematics and statistics to create a mathematical model by learning patterns in a dataset.  The [supervised learning](https://en.wikipedia.org/wiki/Supervised_learning) methods is one of the machine learning categories which dataset includes input and desired outputs/targets/classes/categories. These targets can either have limited set of values or any real value. When the targets are limited set of values, the learning task is known as classification. Each row in the dataset is a sample and the classification is assigning a class label/target to each sample. The mathematical model which are used for this learning task are called classifiers. When the targets are real values, the learning task is called regression and the mathematical model which are used for this task are called regressors. We will go through classification first and look at regression later in this tutorial.